twitter-processor-example/pom.xml

```diff
@@ -25,9 +23,32 @@
   <properties>
     <version.activiti>7-201711-EA-SNAPSHOT</version.activiti>
     <version.activiti.cloud>7-201711-EA-SNAPSHOT</version.activiti.cloud>
+    <version.activiti.cloud.connectors>7-201711-EA-salaboy_1514_twitter_connector-SNAPSHOT</version.activiti.cloud.connectors>
+    <jdk.version>1.8</jdk.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <!-- Plugins non managed -->
+    <maven.docker.plugin.version>0.21.0</maven.docker.plugin.version>
+    <spring-boot-maven-plugin.version>1.5.4.RELEASE</spring-boot-maven-plugin.version>
```

Comment:
Any specific reason to use the `1.5.4.REALEASE` here? I'd expect `2.0.0.M6` instead.